laravel/the-basics/databases.html.md

```diff
@@ -590,6 +597,100 @@ php artisan cache:clear
 
 If all goes well here, you should be good to go! 
 
+## _SQLite in a Laravel Fly App_
+"[SQLite](https://www.sqlite.org/about.html)" is a file-based, lightweight, low-latency, and in-process library providing an embeddable SQL database engine. 
+
+Since SQLite is "embeddable", we no longer have to setup a separate Fly App for it. We simply configure our Laravel Fly App with one:
+
+
+1) Make sure you are in your Laravel Fly App's directory
+
+```cmd
+cd <laravel-fly-app>
+```
+
+2) Create a volume, you'll be attaching this later to your Laravel Fly App's storage directory 
+
+```cmd
+fly volumes create storage_vol --region ams --size 20 #GB
+```
+
+
+3) Revise your Laravel Fly App's `fly.toml` file to connect with the sqlite database and mount the volume created for your storage directory:
+
+```
+[env]
+  APP_ENV = "production"
+  DB_CONNECTION="sqlite"
+  DB_DATABASE=/var/www/html/storage/database/database.sqlite
+
+[mounts]
+  source="storage_vol"
+  destination="/var/www/html/storage"
+```
+
+<aside class="callout">
+Caution! Mounting a Volume to a folder will initially erase any item it contains during the first time the Volume is mounted for the folder. 
+<p></p>
+
+For example, the storage folder contains subfolders app, framework, and logs. 
+Mounting the volume to the storage folder erases these contents, and leaves behind a sole item "lost+found".
+</aside>
+
+3) To fix the little storage-content-erasure issue, please go ahead and make a copy of your storage folder in a dummy folder, storage_.
+
+```cmd
+cp -r storage storage_
+```
+<small>
+ You'll later use this folder to copy over its contents to the volumized storage folder.
+</small>
+
+4) Next create a Start Up Script that will initialize the volumized storage folder's contents.
+
+```cmd
+touch .fly/scripts/1_storage_init.sh
+```
+<small>
+Side Note: Start up scripts are run in numeric-alphabetical order. Naming `1_storage_init.sh` makes sure it is the first script run. Otherwise, naming the file as `storage_init.sh` alone would've moved the `caches.sh` script above it, and would've executed before storage initialization happened. One of the commands in the `caches.sh` will not have worked properly, due to a lack of properly initialized storage directory.
+</small>
+
+<b>On to the content of the Start Up script:</b>
+```bash
+FOLDER=/var/www/html/storage/app
+if [ ! -d "$FOLDER" ]; then
+    echo "$FOLDER is not a directory, copying storage_ content to storage"
+    cp -r /var/www/html/storage_/. /var/www/html/storage
```

Comment:
I would usually do `cp -r /var/www/html/storage_/* /var/www/html/storage` with the `*` instead of `.` - Do you mind checking to see if that's effectively the same thing? I'm not totally sure!

Reply:
(Nice use of startup scripts!)

Reply:
yes sir! I was veery excited to use a new feature!

Reply:
So! I tested 

`cp -r /storage_/* /storage `
and 
`cp -r /storage_/. /other`

All files were copied perfectly the same in both methods! So from test results, practically the same!
<img width="368" alt="image" src="https://user-images.githubusercontent.com/33826441/195178892-71672fa9-1f6b-41f3-85f9-ecf25ebd1974.png">

Reply:
done!